projects/plugins/boost/app/assets/src/js/features/critical-css/show-stopper-error/show-stopper-error.tsx

```diff
@@ -0,0 +1,101 @@
+import { __ } from '@wordpress/i18n';
+import type { CriticalCssState } from '../lib/stores/critical-css-state-types';
+import ErrorNotice from '$features/error-notice/error-notice';
+import { useEffect, useState } from 'react';
+import FoldingElement from '../folding-element/folding-element';
+import ErrorDescription from '../error-description/error-description';
+
+// This simulates a global variable that is shared between all instances of this component.
+let globalValue = true;
+
+const useSharedValue = () => {
+	const [ value, setValue ] = useState( globalValue );
+	const setGlobalValue = newValue => {
+		globalValue = newValue;
+		setValue( newValue );
+	};
+
+	return [ value, setGlobalValue ];
+};
```

Comment:
For cleanliness, I think this should be moved out to a utility library and be called something like `useSingletonState` so that its implementation details can be separated from this. We can then document what singleton state is. :)